modules/standard/Regexp.chpl

```diff
@@ -451,6 +451,11 @@ class BadRegexpError : Error {
                    ``(?U)``.
 
  */
+
+/*
+  throws: SystemError: If CHPL_REGEXP is set to none.
```

Comment:
Unfortunately, this is inaccurate - compilerError is a special function that causes compilation to fail.  It's not something that can be caught by a try/catch block.  You want to look in the body for `throw`.  Looking at the function, it looks like we throw a BadRegexpError.  What can you tell about the circumstances under which that happens?

You'll also want to match the style of other `:throws:` documentation.  Take a look at the documentation comment for modules/standard/Random.chpl's `getNth` for an example of how it should be written.

Reply:
The error is thrown when the variable `CHPL_REGEXP` is set to `none`. The introduction to this file tells that regular expression support is built on the top of the RE2 regular expression library and should be initialized with `re2`. So if its value is `none`, the compiler should throw an error. This is what I have understood after going through the file.

I shall change the error type to BadRegexError as you mentioned.

Thanks for the prompt reply :)